accounts/views.py

```diff
@@ -7,77 +7,93 @@
 
     if request.method == "POST":
         form = CustomUserCreationForm(request.POST)
+        # If signup valid, log user in and redirect to onboarding
         if form.is_valid():
             user = form.save()
             login(request, user)
-            return redirect("dashboard:index")
+            #request.session["onboarding"] = True TODO link to onboarding later
+            #return render(request, "accounts/onboarding.html")
+            return redirect("landing_page:index")
+        # Else, determine error in signup and return message
         else:
             # checks if username is taken
-            username = request.POST["username"]
-            if User.objects.filter(username=username).exists():
+            providedUsername = request.POST["username"]
+            if User.objects.filter(username=providedUsername).exists():
                 return render(
                     request,
-                    "accounts/register.html",
-                    {"form": form, "message": "Username has already been taken"},
+                    "accounts/login.html",
+                    {"form": form, "signUpError": "Username has already been taken"},
                 )
             # checks if email is taken
-            email = request.POST["email"]
-            if User.objects.filter(email=email).exists():
+            providedEmail = request.POST["email"]
+            if User.objects.filter(email=providedEmail).exists():
                 return render(
                     request,
-                    "accounts/register.html",
-                    {"form": form, "message": "Email address is already in use"},
+                    "accounts/login.html",
+                    {"form": form, "signUpError": "Email address is already in use"},
                 )
+            # checks if passwords match
             password1 = request.POST["password1"]
             confirm = request.POST["password2"]
             if password1 != confirm:
                 return render(
                     request,
-                    "accounts/register.html",
-                    {"form": form, "message": "Passwords do not match"},
+                    "accounts/login.html",
+                    {"form": form, "signUpError": "Passwords do not match"},
                 )
-
+            # if all else fails, return unknown error
             return render(
                 request,
-                "accounts/register.html",
-                {"form": form, "message": "An unknown error occured."},
+                "accounts/login.html",
+                {"form": form, "signUpError": "An unknown error occured."},
             )
     elif request.method == "GET":
         form = CustomUserCreationForm()
-        return render(request, "accounts/register.html", {"form": form})
+        return render(request, "accounts/login.html", {"form": form})
+    
+def onboarding(request):
+    #if(request.session.get("onboarding")):
```

Comment:
is this needed?

Reply:
Yeah I commented it out because I wanted to add just the basic user account creation / signout functionality so that anybody working on profile/user specific features can work on it while I do the onboarding stuff